lib/button-svg.js

```diff
@@ -27,9 +27,17 @@ class ButtonSVG {
   }
 
   generateStyle() {
+    const {fontCssUrl, fontFileUrl, fontName} = this.options;
+    const style = (fontCssUrl) ?
```

Comment:
oh... i guess we don't want any font face setting if the url isn't available right?